shuup/utils/multilanguage_model_form.py

```diff
@@ -26,14 +27,18 @@ def to_language_codes(languages):
     return languages
 
 
+def get_default_language():
+    return getattr(settings, "PARLER_DEFAULT_LANGUAGE_CODE", get_language())
```

Comment:
Should we even enforce this value? (`settings.PARLER_DEFAULT_LANGUAGE_CODE`) without the code default is again all over the place, right?